Dockerfile

```diff
@@ -209,4 +209,6 @@ ENV CONFIG_MOUNT_POINT=/boot/config.json \
 	DEFAULT_PUBNUB_SUBSCRIBE_KEY=$DEFAULT_PUBNUB_SUBSCRIBE_KEY \
 	DEFAULT_MIXPANEL_TOKEN=$DEFAULT_MIXPANEL_TOKEN
 
+HEALTHCHECK CMD wget -qO- http://127.0.0.1:${LISTEN_PORT:-48484}/v1/healthy || exit 1
```

Comment:
I added some parameters, do they look like sensible values? With a 5 minute interval and 3 retries, it will take 15 minutes to detect an unhealthy supervisor, which to me feels like a reasonable time to give it a chance to recover / avoid false positives. And the start period of 1 minute should be enough for everything to start up... what do you think?